infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationManager.java

```diff
@@ -204,4 +208,34 @@ public void processStandbyChange(TopologyDescriptor newConfig) {
             }
         }
     }
+
+    /**
+     * Stop the current log replication event and start a full snapshot sync for all standby clusters
+     */
+    public void enforceSnapshotFullSync() {
+        List<UUID> snapshotIds = new ArrayList<>();
+        for (CorfuLogReplicationRuntime standbyRuntime : runtimeToRemoteCluster.values()) {
+            standbyRuntime.getSourceManager().stopLogReplication();
+            UUID snapshotUUID = standbyRuntime.getSourceManager().startSnapshotSync();
+            snapshotIds.add(snapshotUUID);
+            log.info("Enforce Start full snapshot sync for cluster {} with uuid {}", standbyRuntime.getRemoteClusterId(), snapshotUUID);
+        }
+
+        // Block until the full snapshot sync has been started on all the standby clusters.
```

Comment:
Question, what is the idea of blocking? initially it was thought as to not return until it has started, but the forceSnapshotSync from the caller is async and returns right away. I'm not sure I understand how this could help us.